### PR TITLE
Types

### PR DESCRIPTION
**Pull request by ratijas**
**Detailed description**

*  Don't be so hardcore: replace e??~ with e? 
*  Slightly reword on complex types, change example

**Test plan**

None

**Closing issues**

None

### CODE REVIEW

File: src/analysis/types.md [hunk 1]
```diff
@@ -42,21 +42,22 @@ from one platform to another, radare2 uses `definite` types like as
...
+float
 int
 int16_t
 int32_t
-int64_t
-int8_t
-long
-long long
-...
+[...]
```
Comment (trufae) [inline src/analysis/types.md:57]: List them all. Update with the current output

**Review by trufae — CHANGES_REQUESTED** [review]

File: src/analysis/types.md (hunk 1 above)
Comment (ratijas) [reply src/analysis/types.md:57]: For some reason, I have gid_t, uid_t and pid_t out of the box. I'm not sure that's normal, but whatever.

**Comment by trufae** [comment]
So much time without feedback. Are you still wiling to submot more changes in this pr? Or just keep it as is

**Comment by trufae** [comment]
Ping?

**Comment by trufae** [comment]
tired of waiting

**Pull request merged**